client/src/js/services/receipts/ReceiptService.js

```diff
@@ -55,7 +55,8 @@ function ReceiptService($http, util) {
       .then(util.unwrapHttpResponse);
   }
 
-  function patientRegistrations(options) {
+  /** @deprecated */
```

Comment:
This can be removed too.  It will either be done in this PR or in a subsequent one.  Might as well remove this method now.